src/pages/experiments/[experimentId]/plots-and-tables/index.jsx

```diff
@@ -117,39 +15,9 @@ const PlotsTablesHome = ({ experimentId, experimentData }) => {
         experimentData={experimentData}
         title='Plots and Tables'
       />
-      <Space style={{ padding: '0 1em' }} direction='vertical'>
-        {plots.map((section) => (
-          <Row gutter='16'>
-            <Col span={24}>
-              <Divider
-                orientation='left'
-                orientationMargin='0'
-              >
-                <strong>{section.title}</strong>
-              </Divider>
-            </Col>
-            {section.plots.map((item) => (
-              <Col className='plot-card'>
-                <Card
-                  size='small'
-                  hoverable
-                  title={item.name}
-                  bodyStyle={{ padding: '0' }}
-                  style={CARD_STYLE}
-                >
-                  <Link
-                    as={`/experiments/${experimentId}/plots-and-tables/${item.link}`}
-                    href={`/experiments/[experimentId]/plots-and-tables/${item.link}`}
-                    passHref
-                  >
-                    <CardItem item={item} />
-                  </Link>
-                </Card>
-              </Col>
-            ))}
-          </Row>
-        ))}
-      </Space>
+      <SingleTileContainer title='Select Plot'>
```

Comment:
I think the `Select Plot` title is unnecessary. It's just obvious that you have to select a plot and it makes the screen more cluttered imo.

Reply:
Removed